todo_app/static/style.css

```diff
@@ -0,0 +1,18 @@
+.form {
+    
+}
+
+.form_input {
+    border: 0em;
+    margin-right: 0em;
+}
+
+.form_input_button {
+    border: 0em;
+}
+
+.form_input_button:hover {
+    background-color: green;
+    color: white;
+}
+
```

Comment:
you've added some css styles, cool! They're currently not being used because you haven't imported the css into the html file.
To do so, you can add the following line to your `layout.html` file:
`<link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='style.css') }}">` 

(This is optional but thought you might want to see your work!)